src/core/components/layout/components/columns/styles.ts

```diff
@@ -13,32 +14,32 @@ export const columns = css`
 	}
 `;
 
-const collapseBelowSpacing = css`
+const collapseBelowSpacing = (spaceY: ColumnsSpaceY) => css`
 	display: block;
 	& > * {
 		margin-right: 0;
-		margin-bottom: ${space[5]}px;
+		margin-bottom: ${space[spaceY]}px;
 	}
 `;
 
-export const collapseBelowTabletColumns = css`
+export const collapseBelowTabletColumns = (spaceY: ColumnsSpaceY) => css`
 	${until.tablet} {
-		${collapseBelowSpacing}
+		${collapseBelowSpacing(spaceY)}
 	}
 `;
-export const collapseBelowDesktopColumns = css`
+export const collapseBelowDesktopColumns = (spaceY: ColumnsSpaceY) => css`
 	${until.desktop} {
-		${collapseBelowSpacing}
+		${collapseBelowSpacing(spaceY)}
 	}
 `;
-export const collapseBelowLeftColColumns = css`
+export const collapseBelowLeftColColumns = (spaceY: ColumnsSpaceY) => css`
 	${until.leftCol} {
-		${collapseBelowSpacing}
+		${collapseBelowSpacing(spaceY)}
 	}
 `;
-export const collapseBelowWideColumns = css`
+export const collapseBelowWideColumns = (spaceY: ColumnsSpaceY) => css`
 	${until.wide} {
-		${collapseBelowSpacing}
+		${collapseBelowSpacing(spaceY)}
```

Comment:
It feels like there's a nicer way of doing this but I'm not really sure what it is.

Reply:
I've refactored this in https://github.com/guardian/source/pull/776/commits/1a6b4394b939865971a7de60a28fa942661c56f8 so that there's not one function which takes the breakpoint and the `spaceY` value. Something similar could work with the `collapseBelowMap` if this is a pattern that we prefer?